overlord/snapstate/handlers_aliasesv2_test.go

```diff
@@ -801,7 +801,7 @@ func (s *snapmgrTestSuite) TestDoUndoSetAutoAliasesFirstInstallUnaliased(c *C) {
 	c.Assert(err, IsNil)
 
 	c.Check(snapst.AutoAliasesDisabled, Equals, false)
-	c.Check(snapst.AliasesPending, Equals, true)
+	c.Check(snapst.AliasesPending, Equals, false)
```

Comment:
why did this change?

Reply:
Actually after giving this more though it was a bit significant because if setup-aliases failed midway on first install, it would have left those aliases on disk, very rare but possible.